### DISCUSSION TITLE
What are the basics of resolving conflicts and what do I need to keep in mind?

### DISCUSSION THREAD

**Discussion by A-Really-Big-Cat** [bcssov/IronyModManager#273] [Q&A] [answered]
I started messing with Irony yesterday because I heard that merging mods with it can reduce the lag experienced in the species menu (an issue I'm having a lot of problem with). Now the mod load order that I have been using is a bit unorthodox with only minimal sorting, but every time I try to change it substantially in a way that more closely resembles the order recommended by the stellariswiki, something in my game ends up breaking. When I tried to merge the mods into a compressed merge, the ethics and civics become broken. So now I am thinking that before I merge I should first be resolving conflicts between mods, but I am a bit overwhelmed by the interface for doing so. When I click the resolve button, what does the program do and what is being overwritten or changed? What do I need to think about before I resolve a conflict to avoid issues?

**Comment by bcssov** [comment, marked as answer]
If compress merge breaks a mod this means that a mod has an issue itself like a capitalized directory folder.
For example:
`Common\some folder`
While in reality it should be:
`common\some folder`
Zip paths are case sensitive as are other OS file systems. So such a mod usually would not work on MAC\Linux. The only reason why this works on non merged mods on windows is because windows filesystem is not case sensitive.

As for the basics I guess you should take a look at the tutorial videos which explain basics pretty well.

**Reply by A-Really-Big-Cat** [reply]
Ok I'll watch the tutorial videos then.

I couldn't find any capitalized directory folders in my mod list unfortunately, although my search certainly could have missed something. Is there another potential reason? The way you worded it made it seem like there were other possibilities.

**Reply by bcssov** [reply]
That's the only reason that I'm aware of in relation to zip file issues. Aside from that compress merge should be exactly the same as non merged mod collection. Check the error.log for any potential issues.
